doc/classes/EditorExportPlugin.xml

```diff
@@ -159,6 +159,16 @@
 				Return a [PackedStringArray] of additional features this preset, for the given [param platform], should have.
 			</description>
 		</method>
+		<method name="_get_export_option_visibility" qualifiers="virtual const">
+			<return type="bool" />
+			<param index="0" name="platform" type="EditorExportPlatform" />
+			<param index="1" name="option" type="String" />
+			<param index="2" name="advanced_options_enabled" type="bool" />
+			<description>
+				[b]Optional.[/b]
+				Validates [param option] and returns visibility for the specified [param platform]. Default implementation return [code]true[/code] for all options.
```

Comment:
```suggestion
				Validates [param option] and returns visibility for the specified [param platform]. The default implementation returns [code]true[/code] for all options.
```

Reply:
Also I'd add "the" in "returns visibility"